base_multi_search_separator/readme/USAGE.rst

```diff
@@ -0,0 +1,6 @@
+To configure the separator you must:
```

Comment:
better to put this part in a CONFIGURE.rst section.

In the USAGE file you can put a print screen. (something like "before" / "after").